.github/actions/create-release-v1/action.yml

```
@@ -0,0 +1,132 @@
+name: Create release
+description: A GitHub Action to create a production release
+
+inputs:
+  token:
+    description: The GitHub token to use. Defaults to GITHUB_TOKEN
+    default: ${{ github.token }}
+  base:
+    description: The base branch to merge into
+    required: true
+  migrations-dir:
+    description: The directory containing the database migrations e.g. src/database/migrations
+  env-file-path:
+    description: The path to the `.env` file e.g. src/service/sample.env
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        # Fetch all history
+        fetch-depth: 0
```

Should checkout the release branch here

```suggestion
        fetch-depth: 0
        ref: 'release'
```